site/en/_data/home.yml

```diff
@@ -58,7 +58,11 @@ onAndWith:
 posts:
   title: The latest on Chrome OS
   modifiers:
-    shapeOn: 1
+    custom: item-grid--posts
+    subtitle: Explore trends, events, and updates on our community.
+    cta:
+      text: All Articles
+      url: /en/news
```

Comment:
Instead of hard-coding the URl here, because it's always going to point to the locale's `news` page, can we use the available language in `site/_generated/index.njs` to point to the locale-specific version instead?

Reply:
I'm not sure I follow here - could you expand on this?

From my understanding, this sounds like using `locale.code` within the `href` of the anchor where this `cta` object is used and shortening `cta.url` to `news`, e.g: `href="/{{locale.code}}/{{cta.url}}"`?